dts/vendor/nordic/nrf54l_05_10_15.dtsi

```diff
@@ -311,6 +311,7 @@
 				rx-delay-supported;
 				rx-delay = <1>;
 				status = "disabled";
+				cross-domain-default-port = <0x1>;
```

Comment:
Have you considered other encodings? For example to provide `gpio` node reference rather than `port index`.

Actually, if we want to use this `port index` scheme it would be more straight-forward to actually include `index/number` in the property name, like `cross-domain-default-port-index` ?

Also, I think we could drop `cross-domain` part and simply name it `default-gpio-port = <<gpio node reference>>` or `default-gpio-port-index = <<gpio port index>>` so that it could be used for other purposes than this particular cross-domain issue. This would also imply doc alignment in bindings